vm/actor/src/builtin/paych/mod.rs

```diff
@@ -53,22 +56,26 @@ impl Actor {
     /// Resolves an address to a canonical ID address and requires it to address an account actor.
     /// The account actor constructor checks that the embedded address is associated with an appropriate key.
     /// An alternative (more expensive) would be to send a message to the actor to fetch its key.
-    fn resolve_account<BS, RT>(rt: &RT, raw: &Address) -> Result<Address, ActorError>
+    fn resolve_account<BS, RT>(rt: &RT, raw: &Address) -> Result<Address, String>
     where
         BS: BlockStore,
         RT: Runtime<BS>,
     {
-        let resolved = rt.resolve_address(raw)?;
+        let resolved = rt
+            // TODO: fatal error not handled here. To match go this will have to be refactored
+            .resolve_address(raw)
+            .map_err(|e| e.to_string())?
+            .ok_or_else(|| format!("failed to resolve address {}", raw))?;
 
-        let code_cid = rt.get_actor_code_cid(&resolved)?;
+        let code_cid = rt
+            .get_actor_code_cid(&resolved)
+            .expect("Failed to get code Cid")
+            .ok_or_else(|| format!("no code for address {}", raw))?;
 
         if code_cid != *ACCOUNT_ACTOR_CODE_ID {
-            Err(ActorError::new(
-                ExitCode::ErrIllegalArgument,
-                format!(
-                    "actor {} must be an account ({}), was {}",
-                    raw, &*ACCOUNT_ACTOR_CODE_ID, code_cid
-                ),
+            Err(format!(
```

Comment:
It's because in specs-actors there is no actor errors being returned here and no aborts in sub calls. The annoying part is that fatal errors are not handled because it panics internally in the runtime, but that's why I added to TODO to update it to match completely when I make a pass through that actor (wanted to keep scope as contained as possible)